Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1910,6 +1911,17 @@ inline void BuilderCustom::applyValueFontSize(BuilderState& builderState, CSSVal
     builderState.setFontDescription(WTFMove(fontDescription));
 }
 
+inline void BuilderCustom::applyValueFontSizeAdjust(BuilderState& builderState, CSSValue& value)
+{
+    auto fontDescription = builderState.fontDescription();
+    auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
+
+    if (primitiveValue.isNumber() && !primitiveValue.isNegative().value_or(true)) {
```

Comment:
Is a negative number the only kind of invalid number? I’m sort of surprised. I don’t see `isNegative` checks in other style builder functions. Is this sort of unique in that respect?

Reply:
It can be non-negative or none for level 4. Maybe others are trusting that the parser will make the validation before, which we can also trust. But anyway there is a mistake in this code, we are not applying the value if it is a CSSValueNone. I'll fix it.